tests/unit/backend/models/user.server.model.spec.ts

```diff
@@ -145,4 +147,59 @@ describe('User Model', () => {
       )
     })
   })
+
+  describe('Statics', () => {
+    describe('upsertUser', () => {
+      it('should create new User document when user does not yet exist in the collection', async () => {
+        // Arrange
+        const validEmail = `test@${AGENCY_DOMAIN}`
+        await expect(User.countDocuments()).resolves.toEqual(0)
```

Comment:
nit: this syntax is clearer, as we've discussed with yr previously
```
const count = await User.countDocuments()
expect(count).toEqual(0)
```

Reply:
ff0df43